cpp/src/rolling/rolling.cu

```diff
@@ -514,17 +518,19 @@ std::unique_ptr<column> rolling_window_udf(column_view const &input,
   // Launch the jitify kernel
   cudf::jit::launcher(hash, cuda_source,
                       { cudf_types_hpp, cudf_utilities_bit_hpp,
-                        cudf::experimental::rolling::jit::code::operation_h },
+                        cudf::experimental::rolling::jit::code::operation_h,
+                        ___src_rolling_rolling_jit_detail_hpp },
```

Comment:
I think we can get rid of this if `#include "rolling/rolling_jit_detail.hpp"` is in `operation.h` and the `rolling/rolling_jit_detail.hpp` file is added to the JIT file list in `CMakeLists.txt`.

Reply:
`rolling/rolling_jit_detail.hpp` is already in the JIT file list.

Reply:
I misunderstood something about JIT header includes. Carry on!